EventHubs/tests/baseeventhubtest.py

```diff
@@ -83,6 +78,30 @@ def insert_mock_logs_in_EventHub(self, filename):
         sbs.send_event(self.eventhub_name, json.dumps(mock_logs))
 
         print("Event inserted")
+        
+    def insert_mock_metrics_in_EventHub(self, filename):
+        print("Inserting fake metrics in EventHub")
+        
+        defaultauthorule_name = "RootManageSharedAccessKey"
+        namespace_name = self.get_resource_name(self.event_hub_namespace_prefix, "Microsoft.EventHub/namespaces")
+        eventhub_client = EventHubManagementClient(self.azure_credential, self.subscription_id)
+        eventhub_keys = eventhub_client.namespaces.list_keys(self.RESOURCE_GROUP_NAME, namespace_name, defaultauthorule_name)
+        
+        sbs = ServiceBusService(
+            namespace_name,
+            shared_access_key_name=defaultauthorule_name,
+            shared_access_key_value=eventhub_keys.primary_key,
+            request_session=Session()
+        )
+        
+        mock_logs = json.load(open(filename))
```

Comment:
can you please change this as well, this should avoid file not closed morning as seen in the screenshot.
with open(filename, 'r') as template_file_fd:
            template_data = json.load(template_file_fd)

Reply:
change this in line 76 also (insert_mock_logs_in_EventHub function)

Reply:
updated